### DISCUSSION TITLE
Does dependabot run on a schedule AND when dependencies are updated during a merge?

### DISCUSSION THREAD

**Discussion by gmiklich** [community/community#37658] [Code Security]
Just a general question: when you configure the version updates to run on a schedule, Dependabot creates the PRs like you'd imagine.  However, are there other documented or undocumented times when Dependabot will run?  For example, if I decide to batch all of my Dependabot version updates into one PR and merge it, will Dependabot run and notice that there are PRs that no longer update anything and close by themselves?

Along those same lines, is that "auto-closing of no longer relevant PRs" mean that a Dependabot scan for updates has run?  Will new PRs be created for other dependencies then?

Last question that is somewhat related: when a PR limit is reached (the default being 5), and Dependabot "cannot crate any more pull requests", does it queue up the other updates?  Does it check every time a PR is merged if it can open one of the queued ones?  Or if not queued, does closing a PR automatically trigger a Dependabot scan?  Or is it only when closing a PR that was opened by Dependabot that a scan is triggered?

tldr; all the questions boil down to "When does Dependabot scan the dependencies and open or close PRs" because it appears to happen more often than the schedule provided in the yaml file.

Thanks in advance for your time

**Comment by DenverCoder1** [comment]
In addition to the specified frequency, Dependabot runs whenever the Dependabot config changes. It will also run and rebase any open PRs when a package manifest file is changed.

If you update the requirements yourself in a separate PR, Dependabot will run and recheck its open PRs automatically and close that are no longer relevant.

More info: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/about-dependabot-version-updates#frequency-of-dependabot-pull-requests

**Reply by Wallman** [reply]
Thanks! I would love to be able to specify it to **only** run on schedule.
We see that it would work better in an auto-merge context. (To avoid multiple deploys when merging a PR)
